taxi_service/settings.py

```diff
@@ -56,7 +56,7 @@
 TEMPLATES = [
     {
         "BACKEND": "django.template.backends.django.DjangoTemplates",
-        "DIRS": [],
+        "DIRS": [BASE_DIR, "templates"],
```

Comment:
Code Quality Issue: It's not a good practice to hardcode the 'templates' directory in your DIRS setting. Instead, use the os.path or pathlib module to dynamically get the path to the templates directory. This makes your code more portable and easier to maintain.